### PR TITLE
Do not fail if Quay returns status code other than 200

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/prescriptions_refresh/handlers/quay_security.py [hunk 1]
```diff
@@ -192,7 +192,11 @@ def _quay_get_security_info(image_name: str, container_id: str) -> List[Dict[str
         headers={"Authorization": f"Bearer {_QUAY_TOKEN}"},
         params={"vulnerabilities": True},
     )
-    response.raise_for_status()
+    if response.status_code != 200:
+        _LOGGER.exception(
+            f"Failed to obtain security information for image {image_name!r} with container id {container_id!r}"
```
Comment (codificat) [inline thoth/prescriptions_refresh/handlers/quay_security.py:197]: Maybe make it more clear it was an HTTP error / which one?

```suggestion
            f"Failed to obtain security information from {_QUAY_URL} for image {image_name!r} with container id {container_id!r}: HTTP {response.status_code}"
```

**Review by codificat — COMMENTED** [review]
Only a suggestion to add a bit more info to the log message

File: thoth/prescriptions_refresh/handlers/quay_security.py (hunk 1 above)
Comment (fridex) [reply thoth/prescriptions_refresh/handlers/quay_security.py]: Nice catch. I've adjusted the logged error slightly also on another place. PTAL.

**Comment by codificat** [comment]
lgtm, thanks!

Feel free to /approve :)  (I'm not in owners here yet)

**Pull request merged**